WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.Shared/HttpRest/HttpHeadersHelper.cs

```diff
@@ -152,11 +152,11 @@ private void RetrieveCsrfToken(HttpResponseMessage response)
         /// <summary>
         /// Checks a response to see if it failed due to a bad CSRF token.
         /// </summary>
-        /// <param name="response">The response from the REST call.</param>
+        /// <param name="exception">The DevicePortalException from the REST call.</param>
```

Comment:
this pattern feels odd. why not just throw? I see no reason to not have a CsrfTokenException in the project.